client/blocks/login/login-form.jsx

```diff
@@ -614,15 +618,17 @@ export class LoginForm extends Component {
 			return null;
 		}
 
-		const { query, usernameOrEmail } = this.props;
+		const { currentQuery, usernameOrEmail, oauth2Client, gravatarFrom, gravatarFlow } = this.props;
 
 		return getLoginLinkPageUrl( {
 			locale: this.props.locale,
 			currentRoute: this.props.currentRoute,
-			signupUrl: this.props.currentQuery?.signup_url,
-			oauth2ClientId: this.props.oauth2Client?.id,
-			emailAddress: usernameOrEmail || query?.email_address || this.state.usernameOrEmail,
```

Comment:
Fix: [This component uses the `currentQuery`](https://github.com/Automattic/wp-calypso/blob/76759326b6627f278647ed05b95b07c6d8ece2e8/client/blocks/login/login-form.jsx#L1252) but not the `query`.